cli/src/commands/extensions/mod.rs

```diff
@@ -0,0 +1,113 @@
+pub mod extension;
+
+use std::{collections::HashSet, convert::TryFrom, path::PathBuf};
+
+use crate::commands::{CommandResult, CommandValue, ExitCode};
+pub use extension::*;
+use log::{error, info};
+
+use anyhow::Result;
+use clap::{arg, ArgMatches, Command, ValueHint};
+
+pub fn command<'a>() -> Command<'a> {
+    Command::new("extension")
+        .about("Run extensions")
+        .subcommand(
+            Command::new("add")
+                .about("Install extension")
+                .arg(arg!([PATH]).required(true).value_hint(ValueHint::FilePath)),
```

Comment:
The argument here should be a directory, so `ValueHint::DirPath` is more correct.

```suggestion
                .arg(arg!([PATH]).required(true).value_hint(ValueHint::DirPath)),
```